docs/source/troubleshoot.md

```diff
@@ -0,0 +1,23 @@
+<!---
+Copyright 2020 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Troubleshooting
+
+This document is to help find solutions for common problems.
+
+## Firewalled environments
+
+Some cloud and intranet setups have their GPU instances firewalled to the outside world, so if your script is trying to download model weights or datasets it will first hang and then timeout. One possible solution in this situation is to use the ["offline-mode"](https://huggingface.co/transformers/installation.html#offline-mode).
```

Comment:
Could we add the error message encountered? It would be nice if this page was easily searchable by error messages.

Reply:
We haven't heard back from that user back, so we don't know if that is the error message.

But I totally agree that the full error message (perhaps w/o the trace) would be the best.

OK, I use `ufw` to emulate the firewall and I got the exact message as the user. So pasted it.

Thank you!

Reply:
Thanks for adding it!